pyuvsim/GIT_INFO

```diff
@@ -0,0 +1 @@
+["", "", "", ""]
```

Comment:
This should not be added to the repo! If it's showing up it should be added to the .gitignore.